pushapkscript/docker.d/worker.yml

```diff
@@ -55,12 +55,19 @@ products:
               certificate_alias: dep
 
       'COT_PRODUCT == "mobile" && ENV == "prod"':
-        - product_names: [ "fenix"]
+        - product_names: [ "fenix" ]
           digest_algorithm: 'SHA-256'
           skip_check_multiple_locales: true
           skip_check_same_locales: true
           skip_checks_fennec: true
           apps:
+            fennec-nightly:
+              package_names: [ "org.mozilla.fennec_aurora" ]
+              certificate_alias: 'fennec-nightly'
+              google:
+                default_track: 'alpha'
```

Comment:
I was expecting this to be `beta` to match definition from https://github.com/mozilla-releng/scriptworker-scripts/pull/136/files#diff-3ab502ad0f8a21fc4b9a368b6cbcbd8eR37 but I guess it would conflict with vanilla Fenix nightly so it goes to a different channel. Makes sense 👍

Reply:
Ah, good point! I hadn't covered that in our meeting. I'm using `alpha` because QA will have to validate the APK before it reaches regular users. `alpha` is used as a waiting area until then.

Reply:
TIL 👍